src/features/campaigns/store.ts

```diff
@@ -32,5 +36,49 @@ const campaignsSlice = createSlice({
   },
 });
 
+export interface CampaignStoreSlice {
+  callAssignmentList: RemoteList<ZetkinCallAssignment>;
+  surveyList: RemoteList<ZetkinSurvey>;
+}
+
+const initialCampaignState: CampaignStoreSlice = {
+  callAssignmentList: remoteList(),
+  surveyList: remoteList(),
+};
+
+const campaignSlice = createSlice({
+  initialState: initialCampaignState,
+  name: 'campaign',
+  reducers: {
+    callAssignmentCreate: (state) => {
+      state.callAssignmentList.isLoading = true;
+    },
+    callAssignmentCreated: (
+      state,
+      action: PayloadAction<ZetkinCallAssignment>
+    ) => {
+      const callAssignment = action.payload;
+      state.callAssignmentList.isLoading = false;
+      state.callAssignmentList.items.push(
+        remoteItem(callAssignment.id, { data: callAssignment })
+      );
+    },
+    surveyCreate: (state) => {
+      state.surveyList.isLoading = true;
+    },
+    surveyCreated: (state, action: PayloadAction<ZetkinSurvey>) => {
+      const survey = action.payload;
+      state.surveyList.isLoading = false;
+      state.surveyList.items.push(remoteItem(survey.id, { data: survey }));
+    },
+  },
+});
```

Comment:
What's the purpose of this? There is already a store slice for call assignments, and another one for surveys. I see no need for having a store slice called `campaign` where the same data is stored.